lessons/Data-Science/ai-decision-trees/langs/en-us/index.adoc

```diff
@@ -0,0 +1,278 @@
+
+Note that this diagram is *not* a complete decision tree, but a portion of one. The remainder of the tree will grow over the course of the lesson.
+
+Important decision tree conventions:
+
+- Be sure to label arrows with "yes" or "no".
+
+- "Yes" arrows point to the left, and "no" arrows point to the right. (The chain of arrows in the diagram is pointing 45 degrees to the left because both questions were answered "yes".)
+
+As students ask questions, support them in noticing how effective their questions are in sorting the items on the list. "Utensil?" is a great example of a first question, because there are three utensils and three non-utensils on the list of items. "Used to chop?"" would not be a great first question, because you might get lucky and have your answer, but otherwise you still have five items to decide between!
+}
+
+@QandA{
+@Q{How many questions did we ask in order to determine the correct object?}
+@Q{How did we decide which questions to ask?}
+@Q{Which items from our original list (spoon, knife, cup, plate, fork, and mug) did our very first question eliminate from the list of possibilities?}
+@Q{Can anyone think of a different question that would have eliminated more items right off the bat?}
+}
+
+Let's play _another_ round of the game with a new item. We will start with the same first question. As you ask questions, I will add to our previous diagram.
+
+@teacher{
+
+@right{@image{images/tree2.png, 400}}
+
+As students ask questions, add to your diagram.
+
+Round 2 of this game will produce a diagram similar to the one on the right. Notice that the topmost question—"is it a utensil?"—now splits left ("yes, it is a utensil") *and* right ("no, it is not a utensil").
+
+Be sure to always label arrows, as this is a standard convention and improves readability.
+
+}
+
+So far, our diagram offers two unique pathways from the top of the tree to two unique items.
+
+Put another way: __There are two branches that split from the tree's root. Each branch leads to a decision node, which eventually leads to a terminal node (or "leaf").__
+
+@lesson-instruction{
+Let's identify the root, branches, nodes, and leaves on our tree so far.
+}
+
+@teacher{Discuss your in-progress tree to help students locate the different parts. The parts are intuitively named, but formal definitions are below if needed.}
+
+@right{@image{images/terminology-tree.png, 300}}
+
+- The root node is the very top node that represents the entire population or sample.
+- Splitting is the process of dividing a node into sub-nodes with branches.
+- Sub-nodes (or decision nodes) split from the root node, or from other nodes.
+- A leaf node (or terminal node) is a node that does not split.
+
+@vspace{1ex}
+
+@lesson-instruction{
+- With a partner, complete the decision tree on a piece of paper so that all six items are categorized.
+- Now, draw an entirely different decision tree - one which has a __different__ question at its root.
```

Comment:
Suggest: Is "utensil?" the only possible question for the root? Draw a different tree for these same items that has a _different_ question as its root.

Also, why the double underscores?

Reply:
double underscores are from the era when I thought we had to use double underscores (which ended very recently)